components/eam/src/physics/cam/zm_conv.F90

```diff
@@ -2559,25 +3716,34 @@ subroutine cldprp(lchnk   , &
 !
    do i=1,il2g
       doit(i) = .true.
+      totfrz(i)= 0._r8
+      do k = pver,msg + 1,-1
+         totfrz(i)= totfrz(i)+ frz(i,k)*dz(i,k)
+      end do
    end do
    do k=klowest-2,khighest-1,-1
       do i=1,il2g
          if (doit(i) .and. k <= jb(i)-2 .and. k >= lel(i)-1) then
-  	        if (hu(i,k) <= hsthat(i,k) .and. hu(i,k+1) > hsthat(i,k+1) .and. mu(i,k) >= 0.02_r8) then
+            if (hu(i,k) <= hsthat(i,k) .and. hu(i,k+1) > hsthat(i,k+1) .and. mu(i,k) >= 0.02_r8) then
```

Comment:
here's another example of a "magic" threshold buried in the code that would be better to have defined elsewhere along with other constants and thresholds

Reply:
This is not related to this PR. I added a new parameter mu_min=0.02_r8, and replaced 0.02_r8 in the code with mu_min.